panini/nats_client/_asyncio_cli.py

```diff
@@ -72,19 +73,53 @@ async def _establish_connection(self):
             listen_subjects_callbacks = self.listen_subjects_callbacks
             for subject, callbacks in listen_subjects_callbacks.items():
                 for callback in callbacks:
-                    await self.aio_subscribe_new_subject(subject, callback)
+                    await self.aio_subscribe_new_subject(subject, callback, init_subscribtion=True)
 
     def subscribe_new_subject(self, subject: str, callback: CoroutineType):
         self.loop.run_until_complete(self.aio_subscribe_new_subject(subject, callback))
 
-    async def aio_subscribe_new_subject(self, subject: str, callback: CoroutineType):
+    async def aio_subscribe_new_subject(self, subject: str, callback: CoroutineType, init_subscribtion=False):
         wrapped_callback = self.wrap_callback(callback, self)
-        await self.client.subscribe(
+        ssid = await self.client.subscribe(
             subject,
             queue=self.queue,
             cb=wrapped_callback,
             pending_bytes_limit=self.pending_bytes_limit,
         )
+        if not subject in self.ssid_map:
+            self.ssid_map[subject] = []
+        self.ssid_map[subject].append(ssid)
+        if init_subscribtion is False:
+            if not subject in self.listen_subjects_callbacks:
+                self.listen_subjects_callbacks[subject] = []
+            self.listen_subjects_callbacks[subject].append(callback)
+        return ssid
+
+    def unsubscribe_subject(self, topic: str):
+        self.loop.run_until_complete(self.aio_unsubscribe_subject(topic))
```

Comment:
We are not supporting "topic" anymore, use "subject" instead

Reply:
fixed